modules/olm-rukpak-bd.adoc

```diff
@@ -6,13 +6,13 @@
 [id="olm-rukpak-bd_{context}"]
 = BundleDeployment
 
+In {product-title} 4.16, the RukPak `BundleDeployment` API can include an embedded spec for a desired bundle and indicates that it should be active. This includes pivoting from older versions of an active bundle.
```

Comment:
> API can include an embedded spec for a desired bundle

I'm assuming that this was with reference to the BD API where there was a Bundle template embedded (https://github.com/operator-framework/rukpak/blob/0e327c385ca5488da4c2eca95a68d385c7e08a5f/api/v1alpha1/bundledeployment_types.go#L56). This is no longer the case, because BD API will now have all the configuration details, not the bundle template directly.